js/CaTRoX_QWR/Panel_Playlist.js

```diff
@@ -388,13 +388,16 @@ function PlaylistPanel(x, y) {
         var lowerSpace = calcLowerSpace();
         var playlist_w = w - x;
         var playlist_h = Math.max(0, h - lowerSpace - scaleForDisplay(16) - y);
+        var row_h_calc = Math.round(pref.font_size_playlist * 1.667);
 
         this.h = playlist_h;
         this.w = playlist_w;
         this.x = x;
         this.y = y;
 
-        g_properties.row_h = Math.round(pref.font_size_playlist * 1.667);
+        if( row_h_calc > g_properties.row_h ) {
```

Comment:
The problem with this is that if you set your font_size_playlist large, and then set it back smaller, g_properties.row_h will never shrink.

Reply:
One thing that I'd probably be okay with is removing this check, and updating `g_properties.row_h` proportionally at the same time `pref.font_size_playlist` changes. That way it always gets updated, but then if you wanted to, you could increase it in properties yourself. I think that'd be safe.

Reply:
I like that solution better than mine!